aw_core/models.py

```diff
@@ -49,13 +49,13 @@ class Event(dict):
     """
 
     # TODO: Use JSONSchema as specification
-    # FIXME: tags and label have similar/same meaning, pick one
     # FIXME: Some other databases (such as Zenobase) use tag instead of label, we should consider changing
     ALLOWED_FIELDS = {
         "timestamp": datetime,
         "count": int,
```

Comment:
I think count should be removed, better to put such things in the keyvals/data dict.